opentelekomcloud/resource_opentelekomcloud_ces_alarmrule.go

```diff
@@ -334,17 +334,75 @@ func resourceAlarmRule() *schema.Resource {
 	}
 }
 
+func getMetricOpts(d *schema.ResourceData) alarmrule.MetricOpts {
+	mos := d.Get("metric").([]interface{})
```

Comment:
This is probably fine, but may be brittle or cause panics if you get unexpected `nil`s, etc. As long as you are comfortable that you don't need to use the type assertions with the `ok` its fine with me, but just something to be sure about, as panics are obviously very bad for user's state.

Reply:
Done